server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepository.java

```diff
@@ -817,6 +825,7 @@ private static TreeFilter pathPatternFilterOrTreeFilter(@Nullable String pathPat
                 switch (diffEntry.getChangeType()) {
                     case MODIFY:
                         final EntryType oldEntryType = EntryType.guessFromPath(oldPath);
+                        final JsonPatch patch;
```

Comment:
If `patch` is not used outside of `case`, isn't it better to revert and minimize its scope?

Reply:
I'll revert it when when making `Jackson.readTree()` receive `EntryType`.